Sources/PackageModel/Target.swift

```diff
@@ -452,6 +453,7 @@ public final class ClangTarget: Target {
             type: type,
             sources: sources,
             resources: resources,
+            others: others,
```

Comment:
This passes the list of others through for Clang targets as well, which wasn’t being done before.  Warnings were already being emitted for unknown files in Clang targets, but now that we’re doing it later, we have to pass them through (which is correct anyway, otherwise plugins wouldn’t see them for example).  I haven’t found other side effects with this.